app/controllers/save_controller.rb

```diff
@@ -0,0 +1,26 @@
+class SaveController < ApplicationController
```

Comment:
SaveController seems to be a very generic name. Can we be a little specific to not confuse others? can we choose a better name so it is clear to others on what we are doing within this file?

Reply:
Resolved it